server/events.py

```diff
@@ -0,0 +1,98 @@
+from __future__ import annotations
```

Comment:
Isn't it `from __future__ import __annotations__`?

Reply:
Nope, nvm. I thought I did that in my work repo so had to double check :p

Reply:
This gives a `SyntaxError`:
```
SyntaxError: future feature __annotations__ is not defined
```